ExampleCode/C++/README.md

```diff
@@ -81,4 +88,4 @@ You can also just git clone the repository using the following command:
 git clone https://github.com/kazuho/picojson.git
```

Comment:
Further up in this file you mention you have `include.zip` in the "Projects" directory.  Might want to say `Teaching/ExampleCode/C++/Projects/` instead to give the full path.